infra-cdk/lib/stages/ApplicationStage.ts

```diff
@@ -26,6 +26,11 @@ export class ApplicationStage extends cdk.Stage {
             }
         );
 
+        // S3 Bucket for storing images
+        const s3BucketStack = new S3BucketStack(this, `S3BucketStack-${props.stage}-${props.tenant}`, {
+            ...baseStackProps,
+        });
```

Comment:
Nit: could this please be formatted as

```
const s3BucketStack = new S3BucketStack(
  this,
  `S3BucketStack-${props.stage}-${props.tenant}`,
  {
    ...baseStackProps,
  }
);
```

(please set up prettier on your editor for automatic code formatting!)